src/shared/watchdog.c

```diff
@@ -311,12 +311,16 @@ static int watchdog_update_timeout(void) {
 }
 
 static int watchdog_open(void) {
+        static RateLimit watchdog_open_ratelimit = { 5 * USEC_PER_SEC, 1 };
```

Comment:
i think we should lower this to half of the configured ping interval if that's lower, because in that case people apparently want quicker reaction to failures...